packages/eslint-plugin/src/unsupported-route-components.ts

```diff
@@ -7,6 +8,29 @@ function isAllowedElement(name: string) {
   return allowedElements.includes(name)
 }
 
+function checkNodes(
+  nodesToCheck: TSESTree.JSXElement | TSESTree.JSXChild,
+  context: any,
```

Comment:
Is there a better type you could use here?

Reply:
Ah yes, I forgot to take this out. Let me see if I can find the Eslint context.